presidio-anonymizer/presidio_anonymizer/entities/analyzer_results.py

```diff
@@ -0,0 +1,63 @@
+"""Wrapper to the list of AnalyzerResult which manipulates the list."""
+from typing import List
+
+
+class AnalyzerResults(List):
+    """
+    Receives the analyzer result list and manipulates it.
+
+     The manipulation contains removal of unused results and sort by indices order.
+     According to the logic of:
+    - One PII - uses a given or default transformation to anonymize and replace the PII
+    text entity.
+    - Full overlap of PIIs - When one text have several PIIs, the PII with the higher
+    score will be taken.
+    Between PIIs with similar scores, the selection will be arbitrary.
+    - One PII is contained in another - anonymizer will use the PII with larger text.
+    - Partial intersection - both will be returned concatenated.
+    """
+
+    def to_sorted_set(self, reverse=False):
+        """
+        Manipulate the list.
```

Comment:
This line is more confusing than helpful. I suggest to remove it.

Reply:
Can't. linter. changed it.